grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy

```diff
@@ -41,10 +43,14 @@ class PutawayApiController {
             throw new IllegalArgumentException("No putaway found for order ID ${params.id}")
         }
 
+        LocationTypeCode[] locationTypeCodes = LocationTypeCode.BIN_LOCATION
+        List<Location> locations = locationService.getInternalLocations(order.origin, locationTypeCodes, null)
+
         Putaway putaway = Putaway.createFromOrder(order)
         putaway.putawayItems.each { PutawayItem putawayItem ->
```

Comment:
you can add it here and use putaway.origin, it's less confusing
instead of locationTypeCodes, you can use [LocationTypeCode.BIN_LOCATION] directly